src/ethereum_test/types.py

```diff
@@ -290,6 +323,49 @@ class Header:
     base_fee: Optional[int]
     hash: Optional[str] = None
 
+    @staticmethod
+    def from_dict(source: Dict[str, str]) -> "FixtureHeader":
+        """
+        Creates a FixedHeader object from a Dict.
+        """
+        return FixtureHeader(
```

Comment:
Is there a cleaner way of defining this? Sort of difficult to parse visually in this format.

Reply:
I cleaned up a bit, let me know if this looks better. Another option could be to add a method that iterates over the fields.

Reply:
this is much more readable, thx!